.github/act/events/pr_to_main.json

```diff
@@ -0,0 +1,10 @@
+{
+    "pull_request": {
+        "head": {
+            "ref": "build-vendor"
+        },
+        "base": {
+            "ref": "main"
+        }
+    }
+}
```

Comment:
What are these files in act/events used for?

Reply:
These files are placeholder events to simulate the events received by the github CI, so that we can run the CI workflows locally with [act](https://github.com/nektos/act). It's a project to run github actions locally through docker. It has many flaws in my opinion but that's the best local alternative I could find for quickly testing my changes while developing this PR.